src/data/markdown/docs/02 javascript api/07 k6-experimental/01 browser/10 Page/doubledollar--options--.md

```diff
@@ -0,0 +1,30 @@
+---
+title: 'page.$$(selector)'
+excerpt: 'Browser module: page.$$(selector) method'
+---
+
+<Blockquote mod="note" title="">
+
+Use locator-based [`page.locator(selector)`](/javascript-api/k6-experimental/browser/page/locator/) instead.
+
+</Blockquote>
+
+The method finds all elements matching the specified selector within the page. If no elements match the selector, the return value resolves to `[]`.
+
```

Comment:
We should probably have a _Returns_ sub heading in here.

Reply:
Resolve in https://github.com/grafana/k6-docs/commit/95398b184f9afe52df89593693d16dfa2d7085c5